extensions/ql-vscode/src/databases/db-manager.ts

```diff
@@ -18,6 +19,16 @@ export class DbManager {
     });
   }
 
+  public selectedDbItem(): DbItem | undefined {
```

Comment:
Can we name this `getSelectedDbItem` instead? 

I think a general rule we follow is to have verbs in function names. I don't think this is necessarily a hard and fast rule through the codebase, but it's definitely something we follow in this class.

Reply:
Great point! I wondered if it's bad to have two methods with the exact same name but agree that `getSelectedDbItem` makes more sense!